kolibri/core/assets/src/vue/textbox.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+
+  <ui-textbox
+    @focus="$emit('focus')"
+    @input="updateText()"
+    v-model = "currentText"
+    :disabled="disabled"
+    :placeholder = "placeholder"
+    :label = "label"
+    :aria-label = "ariaLabel"
+    :autocomplete = "autocomplete"
+    :autofocus = "autofocus"
+    :required = "required"
+    :type = "type"
+    :invalid="invalid"
+    class="koli-textbox">
+  </ui-textbox>
+
+</template>
+
+
+<script>
+
+  module.exports = {
+    name: 'KolibriTextBox',
+    props: {
+      disabled: {
+        type: Boolean,
+      },
+      autofocus: {
+        type: Boolean,
+      },
+      required: {
+        type: Boolean,
+      },
+      invalid: {
+        type: Boolean,
+      },
+      value: {
+        type: String,
+      },
+      placeholder: {
+        type: String,
+      },
+      label: {
+        type: String,
+      },
+      ariaLabel: {
+        type: String,
+        // enforcing accessibility
+        required: !!this.label,
```

Comment:
I thought it would be cool to make the aria-label required if we have a regular label on the textbox, just as a reminder to use aria-label, essentially. 

Now that I think about it, we might just want to make both of them required, since I can't think of anywhere we'll use a textbox without also using a label.

As for the double-bang, I'm just using that to convert it to a boolean. It seems to work ok just using a string too.